Dockerfiles/Demo

```diff
@@ -29,31 +29,32 @@ RUN set -eux ; \
     \
 # Permission for rootless mode (for running as non-root)
     cd .. ; \
-    chmod 777 logs temp work ; \
-    chmod 777 -R openl-demo
+    touch jetty.state; \
+    chmod 777 logs jetty.state; \
+    chmod 777 -R openl-demo; \
+    \
+# Install fonts required for Apache POI (export into Excel with autowidth of columns)
+    apt-get update ; \
+    apt-get install -y --no-install-recommends \
+    fontconfig ; \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Why do you move it here? As a result **fontconfig** will not be available in the resulted docker image. Because this stage is temporal and used only for unzipping openl-tablets-demo.zip archive. Pay attention to the 59 line. **/demo** folder is just copied from this stage to the resulted one